opencensus/metrics_quickstart/main.go

```diff
@@ -63,23 +63,25 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	// Flush must be called before main() exits to ensure metrics are recorded.
-	defer exporter.Flush()
 
 	if err := exporter.StartMetricsExporter(); err != nil {
 		log.Fatalf("Error starting metric exporter: %v", err)
 	}
-	defer exporter.StopMetricsExporter()
```

Comment:
This call is in the GoDoc for StartMetricsExporter: https://godoc.org/contrib.go.opencensus.io/exporter/stackdriver#Exporter.StartMetricsExporter

Why are we deleting it?

Reply:
I meant to remove only Flush(). I'll add StopMetricsExporter() back